draft-ietf-dnsop-must-not-sha1.md

```diff
@@ -92,9 +92,11 @@ MUST NOT be used when creating DNSKEY and RRSIG records.
 
 Validating resolvers MUST continue to support validation using these
```

Comment:
I think "Validating resolver implementations MUST" ... and "deployed validating resolvers MAY
be configured to continue" . I think that the point was that the SW needs to have support, but people can enable it or not...

Reply:
updated to include your wording suggestions (which I agree with!)